src/lib/crypto/kimchi_bindings/js/node_js/node_backend.js

```diff
@@ -1,29 +1,18 @@
 // Provides: plonk_wasm
-var plonk_wasm = require('./plonk_wasm.js');
-var native = null;
-try {
-  native = require('../native/plonk_napi.node');
-} catch (e) {
-  // native not available, keep WASM
-}
+var plonk_wasm = (function() {
+  var wasm = require('./plonk_wasm.js');
 
-function snakeToCamel(name) {
-  return name.replace(/_([a-z])/g, function (_match, ch) {
-    return ch.toUpperCase();
-  });
-}
+  try {
+    var native =  require('@o1js/native-' + process.platform + '-' + process.arch)
 
-function override(functionName) {
```

Comment:
Don't we prefer a helper here to avoid repeating this pattern over and over again as we have more overrides?